File: C#/Book_Store_C#/CSD221_Lab6_Todd_Pidgeon/Lab6/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab6
{
    /**
    Author : Todd Pidgeon
    Date : 03/06/2017
    Description : the class structure for Ticket.
    **/

    public class Ticket : SaleableItem
    {
        private String description;
        private Double price;
        private String client;

        public double Price
        {
            get
            {
                return price;
            }

            set
            {
                price = value;
            }
        }

        public string Client
        {
            get
            {
                return client;
            }

            set
            {
                client = value;
            }
        }

        public string Description
        {
            get
            {
                return description;
            }

            set
            {
                description = value;
            }
        }

        public double getPrice()
        {
            return Price;
        }

        public void sellCopy()
        {
            throw new NotImplementedException();
        }
    }
}

File: Java/Book_Store/Ticket.java

package Lab4;

/**
Author : Todd Pidgeon
Date : 03/06/2017
Description : the class structure for Ticket.
**/

public class Ticket implements SaleableItem {

    @Override
    public void sellCopy() {
        System.out.println("***TICKET VOUCHER***");
        //throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }

    @Override
    public double getPrice() {
        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }
    
}
